sdk/examples/portal/findBy.ts

```diff
@@ -0,0 +1,5 @@
+import VeraxSdk from "../../src/VeraxSdk";
+
+const veraxSdk = new VeraxSdk(VeraxSdk.DEFAULT_LINEA_TESTNET);
+
+console.log(await veraxSdk.portal.findBy());
```

Comment:
same

Reply:
Don't we want an example to find all `Portals`?